### DISCUSSION TITLE
New Assault Rifle ammo

### DISCUSSION THREAD

**Discussion by OhnoAAAAAAAAAAA** [FakeFishGames/Barotrauma#10781] [Ideas and suggestions]
Assault Rifle feels underwhelming when compared to weapons like Autoshotgun, so why not add new ammo types?
AP/Depleted Fuel ammo
Since Assault Rifle is now a Tier 1 Talent why not add another use for Combat Engineer? Similarly to other Depleted Fuel ammo It has Higher Gunshot Wound damage Less Bleed damage and better Piercing
Hollow Point ammo
This ammo type is similar to the one above but increases bleed damage but decreases Armor Pierce.

**Comment by FourGreenFields** [comment]
AR is not really underwhelming. More material-efficient than anything except the HMG, reasonably cost-efficient, can stunlock, very accurate, has armour-penetration, and second-best damage per inventory slot, after the HMG. DPS and damage/reload is worse than on the autoshotgun, but performance against armour is better in all regards (assuming armour does not get destroyed). On a related note, [pls nurf HMG](https://github.com/Regalis11/Barotrauma/discussions/10773).

That said, I'm in favour of more ammo-types. But not more ammo specifically for the AR, just more ammo types for (nearly) all weapons.
